sdk/tests/types/rent.rs

```diff
@@ -2,21 +2,13 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use iota_sdk::types::block::{
-    output::{Output, Rent, RentStructure},
+    output::{Output, Rent},
     protocol::protocol_parameters,
     rand::output::{rand_account_output, rand_basic_output, rand_foundry_output, rand_nft_output},
 };
 
-const BYTE_COST: u32 = 1;
-const FACTOR_KEY: u8 = 10;
-const FACTOR_DATA: u8 = 1;
-
-fn config() -> RentStructure {
-    RentStructure::new(BYTE_COST, FACTOR_KEY, FACTOR_DATA)
-}
```

Comment:
Seems like the CI fails are due to this? Values are actually different. Maybe just bring it back with a TODO to use default and change tests ?

Reply:
I think it fails because the structs cost more rent now

Reply:
Ok well TODO and/or issue then